docs/src/pages/components/data-grid/localization/localization.md

```diff
@@ -7,14 +7,20 @@ components: DataGrid, XGrid
 
 <p class="description">The Date Grid will support users from different locale, with formating, RTL, and localized strings.</p>
 
-## 🚧 Locale text
+The default locale of Material-UI is English (United States). If you want to use other locales, follow the instructions below.
 
-> ⚠️ This feature isn't implemented yet. It's coming.
->
-> 👍 Upvote [issue #196](https://github.com/mui-org/material-ui-x/issues/196) if you want to see it land faster.
+## Translation keys
+
+You can use the `localeText` prop to pass in your own text and translations.
+You can find all the translation keys supported in [the source](https://github.com/mui-org/material-ui-x/blob/HEAD/packages/grid/_modules_/grid/constants/localeTextConstants.ts) in the GitHub repository.
```

Comment:
I presume we plan to document this properly at some point?

Reply:
@oliviertassinari answered that in one of the comments before - [https://github.com/mui-org/material-ui-x/pull/718#discussion_r545002613](https://github.com/mui-org/material-ui-x/pull/718#discussion_r545002613)

Maybe we can look into documenting it after all the text is converted, although it may be an overkill since it is also in the code and a reference can work just as good.